src/ontology/mondo-edit.obo

```diff
@@ -105869,20 +105869,22 @@ property_value: RO:0002175 NCBITaxon:9940 {source="https://orcid.org/0000-0002-4
 id: MONDO:0005676
 name: borna disease
 def: "An encephalomyelitis of horses, sheep and cattle caused by borna disease virus." [MESH:D001890]
-subset: nord_rare {source="MONDO:NORD"}
-subset: otar {source="MONDO:OTAR"}
-subset: rare
-synonym: "enzootic encephalomyelitis" EXACT [DOID:5154]
-xref: DOID:5154 {source="MONDO:equivalentTo", source="EFO:0007178"}
-xref: EFO:0007178 {source="MONDO:equivalentTo", source="MONDO:EFO"}
-xref: MESH:D001890 {source="MONDO:equivalentTo", source="DOID:5154", source="EFO:0007178"}
+comment: In Mondo we make the distinction between human and non-human animal diseases. Cross-references do not necessarily make this distinction. Annotate to the human or non-human animal term as appropriate. This disease is considered zoonotic. {source="PMID:31924550", source="PMID:34376142"}
+synonym: "enzootic encephalomyelitis" RELATED [DOID:5154]
+xref: DOID:5154 {source="MONDO:relatedTo", source="EFO:0007178"}
+xref: EFO:0007178 {source="MONDO:relatedTo", source="MONDO:EFO"}
+xref: MESH:D001890 {source="MONDO:relatedTo", source="DOID:5154", source="EFO:0007178"}
 xref: SCTID:76851008 {source="DOID:5154"}
-xref: UMLS:C0006023 {source="MONDO:equivalentTo", source="DOID:5154"}
-is_a: MONDO:0005108 {source="DOID:5154", source="MESH:D001890/inferred", source="MONDO:Redundant", source="MONDO:indirect"} ! viral infectious disease
-is_a: MONDO:0005156 {source="EFO:0007178"} ! encephalomyelitis
-intersection_of: MONDO:0005550 ! infectious disease
-intersection_of: MONDO:0100332 NCBITaxon:12455 ! disease has primary infectious agent Borna disease virus
-relationship: disease_has_infectious_agent NCBITaxon:12455 ! Borna disease virus
+xref: UMLS:C0006023 {source="MONDO:relatedTo", source="DOID:5154"}
+is_a: MONDO:0700053 {source="DOID:5154", source="MESH:D001890/inferred", source="MONDO:Redundant", source="MONDO:indirect"} ! viral infectious disease, non-human animal
+is_a: MONDO:1011336 {source="EFO:0007178"} ! nervous system disorder, non-human animal
+intersection_of: MONDO:0700049 {source="PMID:31924550", source="PMID:34376142"} ! infectious disease, non-human animal
+intersection_of: MONDO:0100332 NCBITaxon:12455 {source="PMID:31924550", source="PMID:34376142"} ! disease has primary infectious agent Borna disease virus
```

Comment:
we do not have "sources" for equivalent axioms (I know, it is silly)

Reply:
If you want to keep the PMID, you can also add the relation to the virus taxon as a subclass of, with sources. 
It is a bit redundant, but it's ok

Reply:
Added PMIDs to 'viral encephalitis' subclass.